src/components/shared/layout/Layout.tsx

```diff
@@ -120,7 +121,7 @@ export default function Layout({ children }) {
                 <Sidebar />
                 <div className="h-full w-full flex-1 flex flex-col">
                     <Header />
-                    <div className="block flex-grow h-full w-full bg-gray-100">
+                    <div className={`block flex-grow h-full w-full bg-gray-100 ${currentPage == CurrentPage.NEW_PROJECT ? 'overflow-y-auto' : ''}`}>
```

Comment:
I think we should not change the layout on this level for such specific current page case. Other components also handle this on their own. For example adding "min-h-full h-[calc(100vh-4rem)] overflow-y-auto" to the inner component should have same behaviour

Reply:
changed